docs/pages/desktop-access/rbac.mdx

```diff
@@ -0,0 +1,137 @@
+    # resource has set 'mode: off'.
+    record_sessions:
+      desktop: true
+
+    # Specify whether clipboard sharing should be allowed with the
+    # remote desktop (requires a supported browser). Defaults to true
+    # if unspecified. If one or more of the user's roles has disabled
+    # the clipboard, then it will be disabled.
+    desktop_clipboard: true
+  allow:
+    # Label selectors for desktops this role has access to.
+    windows_desktop_labels:
+      environment: ["dev", "stage"]
+
+    # Windows user accounts this role can connect as.
+    windows_desktop_logins: ["Administrator", "{{internal.windows_logins}}"]
+```
+
+<Admonition type="warning" title="Active Directory Configuration">
+Teleport's RBAC system is not a replacement for proper Active Directory
+administration. Teleport-issued Windows certificates are valid for a small
+amount of time, but they do apply to the entire domain. Proper care should be
+taken to ensure that each Teleport user's roles reflect only the necesary
+Windows logins, and that these Windows users are properly secured.
+</Admonition>
+
+## Labeling
+
+Both `allow` and `deny` rules support `windows_desktop_labels` selectors. These
+selectors are matched against the labels set on the desktop. It is possible to
+use wildcards (`"*"`) to match all desktop labels.
+
+Windows desktops acquire labels in two ways:
+
+1. The `host_labels` rules defined in the Windows Desktop Service configuration
+2. Automatic `teleport.dev/` labels applied by Teleport (for desktops discovered
+   via LDAP only)
+
+For example, the following `host_labels` configuration would apply the
+`environment: dev` label to a Windows desktop named `test.dev.example.com`,
+and the `environment: prod` label to `desktop.prod.example.com`:
+
+```yaml
+  host_labels:
+  - match: '^.*\.dev\.example\.com$'
+    labels:
+      environment: dev
+  - match: '^.*\.prod\.example\.com$'
+    labels:
+      environment: prod
+```
+
+For desktops discovered via LDAP, Teleport applies the following labels automatically:
+
+| Label | LDAP Attribute | Example |
+| ----- | -------------- | ------- |
+| `teleport.dev/computer_name` | `name` | `WIN-I5G06B8RT33`
+| `teleport.dev/dns_host_name` | [`dNSHostName`](https://docs.microsoft.com/en-us/windows/win32/adschema/a-dnshostname) | `WIN-I5G06B8RT33.example.com`
+| `teleport.dev/os` | [`operatingSystem`](https://docs.microsoft.com/en-us/windows/win32/adschema/a-operatingsystem) | `Windows Server 2012`
+| `teleport.dev/os_version`| [`osVersion`](https://docs.microsoft.com/en-us/windows/win32/adschema/a-operatingsystemversion) | `4.0`
+| `teleport.dev/windows_domain`| Sourced from config | `example.com`
+| `teleport.dev/is_domain_controller` | `primaryGroupID` | `true`
+
+## Logins
+
+The `windows_desktop_logins` role setting lists the Windows user accounts that
+the role permits access to. Like with SSH access, the
+`{{internal.windows_logins}}` variable can be used for local users, which maps
```

Comment:
Does "which maps" modify "local users" here? Should this be "which map"?

Reply:
I think this is correct as written.

Reply:
Is another wording of this sentence "...local users maps to any logins..."? I may not be reading it correctly.

In any case, I think we should rework the sentence beginning "Like with SSH access" so users don't need to understand our Server Access documentation before they can follow this guide.